scripts/python/tao_build.py

```diff
@@ -342,6 +342,8 @@ def _write(line, cmd="build"):
         _write("--host_jvm_args=-Djdk.http.auth.tunneling.disabledSchemes=", cmd = "startup")
         logger.info("configuring tao_bridge with bazel ......")
 
+    with cwd(root):
+        execute('rm -f tao_compiler/decoupling/version.h')
```

Comment:
Why deleting version.h here?

Reply:
+1

Reply:
if `version.h` exists (i.e. re-configure project), it will fail at L355, so delete it in advance.

Reply:
anyway, I removed this explicit `rm -f` and use force copy on L354.